packages/cli/src/server/util/messageSocket.js

```diff
@@ -232,7 +232,4 @@ function attachToServer(server, path) {
   };
 }
 
-module.exports = {
-  attachToServer,
-  parseMessage,
-};
+export default { attachToServer, parseMessage };
```

Comment:
while we're at, can you make these named exports instead of default?

Reply:
@thymikee Yes I can, but when I was working at it, I noticed:
```
  const wsProxy = webSocketProxy.attachToServer(
    serverInstance,
    '/debugger-proxy'
  );
  const ms = messageSocket.attachToServer(serverInstance, '/message');
```
( runServer.js )

so we can end up having 2 packages with `attachToServer`

I kept them as `default` to avoid a collision. But I can name it `attachToMessageSocket`?

Reply:
Would be great to open an issue tracking this. I would like to move to refine exports at some point to embrace this module system even further.

Reply:
We could leverage @Trancever work on #145 for that 🙂